src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/view-data-brokers/View.tsx

```diff
@@ -5,63 +5,77 @@
 import styles from "../dataBrokerProfiles.module.scss";
 import { getL10n } from "../../../../../../../../functions/server/l10n";
 import { DataBrokerProfiles } from "../../../../../../../../components/client/DataBrokerProfiles";
-import { LatestOnerepScanData } from "../../../../../../../../../db/tables/onerep_scans";
 import { AboutBrokersIcon } from "./AboutBrokersIcon";
 import { Button } from "../../../../../../../../components/server/Button";
+import { FixView } from "../../FixView";
+import {
+  StepDeterminationData as StepDeterminationData,
+  getNextGuidedStep,
+} from "../../../../../../../../functions/server/getRelevantGuidedSteps";
 
 export type Props = {
-  scanData: LatestOnerepScanData;
+  data: StepDeterminationData;
+  subscriberEmails: string[];
 };
 
 export const ViewDataBrokersView = (props: Props) => {
   const l10n = getL10n();
 
-  const countOfDataBrokerProfiles = props.scanData.results.length;
+  const countOfDataBrokerProfiles =
+    props.data.latestScanData?.results.length ?? 0;
+
+  const stepAfterSkip = getNextGuidedStep(props.data, "Scan");
 
   return (
-    <div>
-      <div className={styles.content}>
-        <h3>
-          {l10n.getString(
-            "fix-flow-data-broker-profiles-view-data-broker-profiles-headline",
-            { data_broker_sites_results_num: countOfDataBrokerProfiles }
-          )}
-        </h3>
-        <p>
-          {l10n.getString(
-            "fix-flow-data-broker-profiles-view-data-broker-profiles-content"
-          )}
-        </p>
-      </div>
-      <div className={styles.content}>
-        <h4 className={styles.questionTooltipWrapper}>
-          {l10n.getString(
-            "fix-flow-data-broker-profiles-view-data-broker-profiles-view-info-on-sites"
-          )}
-          <AboutBrokersIcon />
-        </h4>
-        <DataBrokerProfiles data={props.scanData.results} />
-      </div>
-      <div className={styles.buttonsWrapper}>
-        <Button
-          variant="primary"
-          href="/redesign/user/dashboard/fix/data-broker-profiles/automatic-remove"
-        >
-          {l10n.getString(
-            "fix-flow-data-broker-profiles-view-data-broker-profiles-button-remove-for-me"
-          )}
-        </Button>
-        <Button
-          variant="secondary"
-          href={
-            "/redesign/user/dashboard/fix/data-broker-profiles/manual-remove"
-          }
-        >
-          {l10n.getString(
-            "fix-flow-data-broker-profiles-view-data-broker-profiles-button-remove-manually"
-          )}
-        </Button>
+    <FixView
+      data={props.data}
+      subscriberEmails={props.subscriberEmails}
+      nextStepHref={stepAfterSkip?.href ?? ""}
+    >
+      <div>
```

Comment:
All the content in here is the same except for additional indenting, but now wrapped in `<FixView>`, and with the data passed in as props.